panda/html/index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="kor">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>판다마켓</title>
+    <link rel="stylesheet" href="../css/style.css">
+    <link rel="icon" href="../img/randing/favicon.ico">
+    <link href="https://fastly.jsdelivr.net/gh/Project-Noonnu/noonfonts_2107@1.1/Pretendard-Regular.woff">
+</head>
+<body>
+    <header>
+        <div class="header-top">
+            <a href="../html/index.html">
+                <img class="logo" src="../img/randing/logoimg.png">
+            </a>
+            <a class="btn" href="../html/login.html">로그인</a>
+        </div>
+        <div class="header-bg">
+            <h1>일상의 모든 물건을<br>거래해 보세요</h1>
+            <a class="btn" href= "items.html">구경하러 가기</a>
+        </div>
+    </header>
+    <main>
+        <section>
+            <div class="tab">
+                <img src="../img/randing/home_1.png" alt="hot item">
+                <div  class="tab-txt">
+                    <h5>Hot item</h5>
+                    <h1>인기 상품을 <br> 확인해 보세요</h1>
+                    <p>
+                        가장 HOT한 중고거래 물품을<br>
+                        판다마켓에서 확인해 보세요
+                    </p>
```

Comment:
`h` 태그를 사용할때는 헤드라인의 의미로 구분을 해야합니다.  즉 `h1`이 `h5` 보다 먼저 나와야하고 
한 페이지에는 여러 `h1`이 나오는것은 적절하지 못합니다. 

```suggestion
                    <h2>Hot item</h2>
                    <p>인기 상품을 <br> 확인해 보세요</p>
                    <p>
                        가장 HOT한 중고거래 물품을<br>
                        판다마켓에서 확인해 보세요
                    </p>
```